### PR TITLE
feat(multi language and small feature): Check if password is valid on test connection

### PR DESCRIPTION
**Pull request by Muurilo**
Multi language support implemented and now when you will test the connection if the password is
wrong, it will return a toast

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -13,6 +13,8 @@ import { GlobalStyle } from './styles/GlobalStyle'
 import { Container, Content } from './styles/Main'
 import { defaultTheme } from './styles/theme'
 
+import '../i18n.js'
```
Comment (diego3g) [inline src/App.tsx:16]: Why are you using js instead of ts here?

File: src/components/NewConnectionModal/index.tsx [hunk 2]
```diff
@@ -111,11 +113,20 @@ const NewConnectionModal: React.FC<ModalProps> = ({
...
-          type: 'error',
-          title: 'Error on connection',
-          description: 'Error estabilishing connection with your Redis server'
-        })
+        const message = err.message
+        if (message.search('WRONGPASS')) {
+          addToast({
+            type: 'error',
+            title: 'Error on connection',
+            description: 'Damn it, wrong password maybe 1234?'
```
Comment (diego3g) [inline src/components/NewConnectionModal/index.tsx:121]: I think that `'Damn it, wrong password! Maybe 1234?'` would be better.

Comment (Muurilo) [reply src/components/NewConnectionModal/index.tsx:121]: Okay

File: src/App.tsx (hunk 1 above)
Comment (Muurilo) [reply src/App.tsx:16]: idk i will change

Comment (Muurilo) [reply src/App.tsx:16]: I couldn't find the @types for i18next-fs-backend

Comment (Muurilo) [reply src/App.tsx:16]: i18next-fs-backend not have a types

**Comment by Muurilo** [comment]
Updated

**Comment by Muurilo** [comment]
how should i proceed since the package does not have the types?

**Comment by natanaelsirqueira** [comment]
@Muurilo You could try using [i18next-http-backend](https://github.com/i18next/i18next-http-backend).

**Comment by Muurilo** [comment]
I believe that with electron it is not possible, the examples I saw it was necessary to use the backend with FS because electron does not create an http server

**Comment by natanaelsirqueira** [comment]
Right. Then maybe try creating an issue for adding type definitions in `i18next-fs-backend`? I think it would very similar to the ones in `i18next-http-backend`.

**Comment by Muurilo** [comment]
I'm going to create an issue thanks for the idea, I already tried to create it myself and it didn't work.

**Comment by diego3g** [comment]
Sorry for the delay, if the lib does not have typings, we need to create them or use another library. I think it's not worth to merge this feature without this, do you agree?

**Comment by Muurilo** [comment]
I agree, I will solve this problem I already created an issue in the package.

**Comment by Muurilo** [comment]
I found a solution, I did a lot of research and there is no way to replace the backend package so this is the only possible solution for that, typescript will no longer look for the typing of the backend fs and also of the language detector, I thought well and this is not will be missed.

**Comment by diego3g** [comment]
Can you fix the conflicts?

**Comment by Muurilo** [comment]
Conflicts fixed!

**Comment by Muurilo** [comment]
Now i18next starts up and works correctly, all you need to do is translate the application

**Pull request merged**